lib/liquid/forloop_drop.rb

```diff
@@ -0,0 +1,40 @@
+module Liquid
+  class ForloopDrop < Drop
+    def initialize(name, length, parentloop)
+      @name = name
+      @length = length
+      @parentloop = parentloop
+      @index = 0
+    end
+
+    attr_reader :name, :length, :parentloop
+
+    def increment
```

Comment:
hm if I make it protected, I can't call it from the `For` class either though :-/